packages/realm-app-importer/src/AppConfigBuilder.ts

```diff
@@ -18,6 +18,7 @@
 
 export type AppConfig = {
   name: string;
+  baseFilePath?: string;
```

Comment:
I (still) don't think this line is needed.

Reply:
You're right. I reworked it (not using it) but forgot remove it.